util/helm/cmd.go

```diff
@@ -60,10 +60,6 @@ func (c Cmd) run(args ...string) (string, error) {
 			fmt.Sprintf("HELM_HOME=%s", c.helmHome))
 	}
 
-	if c.IsHelmOci {
```

Comment:
Please keep it for now and create an issue 2.4 milestone issue to remove it. Some users might want to "downgrade" helm to using custom tooling. If we remove `HELM_EXPERIMENTAL_OCI` it will be impossible.

Reply:
Updated the PR to keep the variable